From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Fri, 18 Dec 2020 02:43:11 +0000
Subject: [PATCH] Fix compability with autoconf 2.70

autoreconf from autoconf 2.70 requires that AM_GNU_GETTEXT_VERSION (or
AM_GNU_GETTEXT_REQUIRE_VERSION) be defined at toplevel in configure.ac
whenever AM_GNU_GETTEXT is used.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 4e4c2b1..f432002 100644
--- a/configure.ac
+++ b/configure.ac
@@ -20,6 +20,7 @@ GETTEXT_PACKAGE=xdg-user-dirs
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [The gettext domain name])
 AC_SUBST(GETTEXT_PACKAGE)
 
+AM_GNU_GETTEXT_REQUIRE_VERSION([0.20.1])
 AM_GNU_GETTEXT([external])
 
 dnl ==========================================================================
